Cargo.toml

```diff
@@ -32,7 +32,9 @@ unsafe_flight = ["abomonation", "abomonation_derive", "arcon_macros/unsafe_fligh
 kafka = ["rdkafka", "futures", "serde_json", "serde"]
 thread_pinning = ["kompact/thread_pinning"]
 socket = ["tokio-util", "futures", "serde_json", "serde"]
-metrics = []
+arcon_metrics=["metrics", "metrics-exporter-prometheus", "metrics-util"]
+hardware_counters=["perf-event", "arcon_metrics"]
```

Comment:
Not sure if there is a formatter for Cargo.toml, but could u fix the spacing? e.g., hardware_counters = ["..."] and not hardware_counters=[".."]